src/state/slices/conversationsSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { conversationsFulfilled } from 'services/conversations/conversations';
+
+const initialState = {
+  conversations: [],
```

Comment:
No need to actually store the conversations, targets and topics. You can use **tags**, which cache these things and allow to re-fetch i.e. after deleting a target.
https://redux-toolkit.js.org/rtk-query/usage/automated-refetching